tests/drop_packets/test_configurable_drop_counters.py

```diff
@@ -248,7 +248,8 @@ def mock_server(fanouthosts, testbed_params, arp_responder, ptfadapter, duthost)
                                 server_dst_addr,
                                 MOCK_DEST_IP)
     _send_packets(duthost, ptfadapter, pkt, server_dst_port, count=100)
-
+    # Issue a ping to populate ARP table on DUT
+    duthost.command('ping %s -c 3' % server_dst_addr, module_ignore_errors=True)
```

Comment:
Is the `send_packets` above still needed with this addition? :)

Reply:
Thanks Danny. The send_packes is not needed. I've made an update and verified.

Reply:
Thanks, Bing! I think `hex_to_mac` also becomes unnecessary with that change.

Reply:
Indeed. Updated again. Thanks!